javaslang/src/main/java/javaslang/Value.java

```diff
@@ -516,7 +516,7 @@ default CharSeq toCharSeq() {
      */
     @SuppressWarnings("unchecked")
     @GwtIncompatible("reflection is not supported")
-    default T[] toJavaArray(Class<T> componentType) {
+    default T[] toJavaArray(Class<? super T> componentType) {
```

Comment:
> I think we should only revert, if we can find a better solution...

Definitely not. We will not introduce unsafe/breaking code. A better solution would be to deprectate (and remove in future) the whole method if it makes no sense.